Project.swift

```diff
@@ -9,17 +10,10 @@ extension Project {
             name: name,
             organizationName: "Nimble",
             targets: [
-                Target(
-                    name: name,
-                    platform: .iOS,
-                    product: .app,
-                    bundleId: "co.nimblehq.\(name)",
-                    infoPlist: .extendingDefault(with: [
-                        "UILaunchScreen": [:]
-                    ]),
-                    sources: ["Sources/**"],
-                    resources: ["Resources/**"]
-                )
+                // Need to update "ProjectName" to `name` when implement script
+                Target.mainTarget(name: "ProjectName"),
```

Comment:
I think it can take the value of parameter `name`
`Target.mainTarget(name: name)`

Reply:
@phongvhd93 The folder name needs to update to the project name but we haven't implemented a script to do that yet. So I leave the folder name to be something like `ProjectName` and `ProjectNameTests` first. 
When initializing targets it needs to use `ProjectName` first to find the correct path otherwise it will crash. 😅